openedx/core/djangoapps/user_api/tests/test_constants.py

```diff
@@ -228,7 +228,7 @@
     ("TO", "Tonga"),
     ("TT", "Trinidad and Tobago"),
     ("TN", "Tunisia"),
-    ("TR", "Turkey"),
+    ("TR", "Türkiye"),
```

Comment:
Turkiye changed its name to avoid references to :turkey:. United Nations [recognized](https://www.un.org/en/about-us/member-states/turkiye) the change, so it is fine to do this change.